eval/eval.py

```diff
@@ -942,9 +1621,13 @@ def token_results(hyp_annotations: List[gc.Annotation], ref_annotations: List[Er
 
     # This return value will be pickled and sent back to the parent process
     return dict(
-        stats=stats, true_positives=true_positives, false_negatives=false_negatives, ups=ups
+        stats=stats, true_positives=true_positives, false_negatives=false_negatives, ups=ups, errtypefreqs=errtypefreqs
     )
 
+def baseline():
+    """ Get a baseline for error detection; nothing is detected """
+
```

Comment:
Something missing? At least `pass`

Reply:
Remnants of experimentation that didn't work out, deleted.